man/shadow.5.xml

```diff
@@ -199,22 +192,21 @@
 	</term>
 	<listitem>
 	  <para>
-	    The date of expiration of the account, expressed as the number
-	    of days since 1970-01-01 00:00:00 UTC.
+	    The date when the account expires,
+	    expressed as the number of days since 1970-01-01 00:00:00 UTC.
 	  </para>
 	  <para>
-	    Note that an account expiration differs from a password
-	    expiration.  In case of an account expiration, the user shall
-	    not be allowed to login.  In case of a password expiration,
-	    the user is not allowed to login using her password.
+	    Note that an account expiration differs from a password expiration.
+	    Account expiration prevents the user from logging in,
+	    whereas password expiration only prevents users login with their password.
 	  </para>
 	  <para>
-	    An empty field means that the account will never expire.
+	    An empty field means that the account never expires.
 	  </para>
 	  <para>
-	    The value 0 should not be used as it is interpreted as either
-	    an account with no expiration, or as an expiration on Jan 1,
-	    1970.
+	    The value 0 should not be used,
+	    as it is interpreted either as an account with no expiration
+	    or as an expiration date of 1970-01-01 00:00:00 UTC.
```

Comment:
[Not a comment about your changes, but to the existing text too.]

Hmmm, this is interesting.  How can a 0 be interpreted as two different things?  We should check which is the behavior.  Am I missing something?

Cc: @hallyn , @ikerexxe

Reply:
Makes sense.  Thanks!